backend/internal/userschema/store/queries.go

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (c) 2025, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+// Package store defines the database queries for user schema management operations.
+package store
+
+import "github.com/asgardeo/thunder/internal/system/database/model"
+
+var (
+	// QueryGetUserSchemaCount retrieves the total count of user schemas.
+	QueryGetUserSchemaCount = model.DBQuery{
+		ID: "ASQ-USER_SCHEMA-001",
+		PostgresQuery: `
```

Comment:
Shall we check for formatting? Most of these lines should fit into a single line